Training/Model/Cow.cs

```diff
@@ -1,7 +1,14 @@
 ﻿namespace Training.Model {
     public class Cow: BaseEntity {
+        
         public required string Name { get; set; }
         public int Age { get; set; }
         public string? Color { get; set; }
+        public string? Description { get; set; }
+        = string.Empty;
```

Comment:
why is this needed?

Reply:
to make it empty by default in case the description is not added

Reply:
then remove ?